Sources/MapboxNavigation/NavigationCameraStateTransition.swift

```diff
@@ -102,6 +107,16 @@ public class NavigationCameraStateTransition: CameraStateTransition {
     }
     
     public func updateForFollowing(_ cameraOptions: CameraOptions) {
+        // Check whether the original and target locations and headings are larger than a certain threshold when current camera state is following.
+        guard let mapView = mapView,
+              let center = cameraOptions.center,
+              let zoom = cameraOptions.zoom,
+              let bearing = cameraOptions.bearing else { return }
+
+        let centerUpdateThreshould = baseCenterUpdateThreshould * pow(2, 22.0 - Double(zoom))
+        guard mapView.cameraState.center.distance(to: center) > centerUpdateThreshould,
```

Comment:
`LocationCoordinate2D.distance(to:)` returns a physical distance in meters, whereas `centerUpdateThreshold` appears to be expressed in screen points.

Reply:
Thanks for the suggestion, I'll add the `getMetersPerPixelAtLatitude(_:_:)` for the physical distance in meters, and the pixel threshold as 5.